.github/workflows/build-test.yml

```diff
@@ -0,0 +1,45 @@
+name: tests
```

Comment:
I cannot see the linters in the pipeline.

Reply:
I suggest we do this in a next PR. Will create a new issue